#!/usr/bin/env python
PACKAGE = "project1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

size_enum = gen.enum([gen.const("Euler",int_t,0,"Euler integration"),
                    gen.const("RK",int_t,1,"Runge-Kutta integration")], "enumeration used to set the type of integration of the odometry")

gen.add("odometry_integration_type",int_t,2,"the integration type parameter that is modified via an enumeration",0,0,1,edit_method=size_enum)

exit(gen.generate(PACKAGE,"project1","dynamic_rec"))